### PR TITLE
[14.0][FIX] shopfloor_single_product_transfer - set destination before posting

### PR DESCRIPTION
**Pull request by mmequignon**
We have to set the destination before posting the move.

ping @jbaudoux @mt-software-de 

customer ref: 11061

### CODE REVIEW

File: shopfloor_single_product_transfer/tests/test_set_quantity.py [hunk 1]
```diff
@@ -750,6 +750,9 @@ def test_set_quantity_scan_package_not_empty(self):
             message=expected_message,
             popup=expected_popup,
         )
+        self.assertEqual(package)
```
Comment (TDu) [inline shopfloor_single_product_transfer/tests/test_set_quantity.py]: ?

Comment (mmequignon) [reply shopfloor_single_product_transfer/tests/test_set_quantity.py]: uh ?

Comment (mmequignon) [reply shopfloor_single_product_transfer/tests/test_set_quantity.py]: fixed

**Review by TDu — APPROVED** [review]

**Review by mt-software-de — APPROVED** [review]
LGTM

**Review by jbaudoux — APPROVED** [review]

**Pull request merged**